### ISSUE TITLE
How to create a new TLS/SSL port?

### ISSUE THREAD

**Issue by vitorgatti** [Snipa22/nodejs-pool#359]
So I tried connecting xmr-stak to the default SSL 9000 port in the pool, but it doesn't work (SOCKET ERROR).

How can I create a new TLS/SSL port for my pool?

Thanks

**Comment by bobbieltd** [comment]
Try with claymore

**Comment by vitorgatti** [comment]
Yes it works with claymore, but xmr-stak has TLS/SSL support, I'd like that my pool supported that feature :P
